src/@next/Icon/icons/icons.ts

```diff
@@ -59,6 +59,8 @@ export const iconNames = [
   'ri-facebook-circle-line',
   'ri-file-fill',
   'ri-file-line',
+  'ri-file-list-3',
+  'ri-folder-user',
```

Comment:
```suggestion
  'ri-folder-user-fill',
```

same with this one 🙇